doc/ReconstructingMeasurements.rst

```diff
@@ -0,0 +1,74 @@
+###############################
+Investigating LSST Measurements
+###############################
+
+When attempting to understand measurements made by the LSST stack, it is often
```

Comment:
Added what I hope is a better description for what is going on with the exposure